src/constants/properties.ts

```diff
@@ -0,0 +1,3 @@
+export enum Properties {
+  ID = 'id',
```

Comment:
@gerimujo496 What do properties represent? Are these properties related to models? If yes, I would suggest separate enums for each resource/model.